<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CashierWindow</class>
 <widget class="QMainWindow" name="CashierWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="cashierStack">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>821</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>821</width>
        <height>451</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>20</y>
            <width>91</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;About Me&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QStackedWidget" name="homeStack">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>391</width>
            <height>391</height>
           </rect>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_8">
           <widget class="QPushButton" name="editCashierButton">
            <property name="geometry">
             <rect>
              <x>208</x>
              <y>300</y>
              <width>171</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>Change Password</string>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget_3">
            <property name="geometry">
             <rect>
              <x>8</x>
              <y>40</y>
              <width>91</width>
              <height>211</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Mobile No.:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>E-mail ID:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget_2">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>40</y>
              <width>281</width>
              <height>211</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="cashierNameLabel">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cashierMobileLabel">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="cashierEmailLabel">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QLabel" name="cashierUsernameLabel_2">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>10</y>
              <width>141</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_9">
           <widget class="QLineEdit" name="oldPassword">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>50</y>
              <width>271</width>
              <height>25</height>
             </rect>
            </property>
            <property name="placeholderText">
             <string>Give your old Password</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="newPassword">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>100</y>
              <width>271</width>
              <height>25</height>
             </rect>
            </property>
            <property name="placeholderText">
             <string>Give New Password</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="confirmPassword">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>150</y>
              <width>271</width>
              <height>25</height>
             </rect>
            </property>
            <property name="placeholderText">
             <string>Confirm your password</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_9">
            <property name="geometry">
             <rect>
              <x>250</x>
              <y>220</y>
              <width>89</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_11">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>220</y>
              <width>89</width>
              <height>25</height>
             </rect>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>20</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;About Company&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget_4">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>60</y>
            <width>321</width>
            <height>341</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="nameLabel_2">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="vatLabel">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="phoneLabel_2">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="addressLabel">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="emailLabel_2">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="websiteLabel">
             <property name="text">
              <string>No</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>71</width>
            <height>341</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>VAT No.:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Phone:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Address:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>E-mail ID:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Website:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;HOME&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>821</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="allButton">
         <property name="text">
          <string>All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="groceryButton">
         <property name="text">
          <string>Grocery</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fashionButton">
         <property name="text">
          <string>Fashion</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="homeappButton">
         <property name="text">
          <string>Home Appliances</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="kidButton">
         <property name="text">
          <string>Kids</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="liquorButton">
         <property name="text">
          <string>Liquor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="giftsButton">
         <property name="text">
          <string>Games</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="otherButton">
         <property name="text">
          <string>Others</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="productTable">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>591</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>671</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Today</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Yesterday</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>30</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>History</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>768</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="pic">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>26</y>
       <width>101</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>Pic</string>
     </property>
    </widget>
    <widget class="QLabel" name="cashierUsernameLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>101</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="homeButton">
        <property name="text">
         <string>Home</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="productButton">
        <property name="text">
         <string>Product</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saleButton">
        <property name="text">
         <string>Sale</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="logoutButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>490</y>
       <width>99</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>120</y>
      <width>371</width>
      <height>521</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>530</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;POWERED BY&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;SUPERMARKET BILLING SUSTEM&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;BILLING&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="productId">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Product ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Add More</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Print Bill</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>291</width>
       <height>231</height>
      </rect>
     </property>
     <widget class="QWidget" name="page_4">
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>91</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>Ok</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_8">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5"/>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>1238</x>
      <y>650</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Account</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
